File: vue-backend-websockets/src/main/java/com/vuebackend/communication/DataPoint.java
package com.vuebackend.communication;

public class DataPoint{

    private double value;
    private long time;


    public DataPoint(double value) {
        this.value = value;
        this.time = System.currentTimeMillis();
    }

	public double getValue() {
		return value;
	}

	public long getTime() {
		return time;
	}
}

File: vue-webapp/src/classes/DataPoint.ts
export default class Datapoint {
    
    private value: number;
    private time: number;
    

    public constructor(value?: number) {
        if(value != null) {
            this.value = value;
        }
    }

    public getValue() : number {
        return this.value;
    }

    public getTime() : number {
        return this.time;
    }
}